package.json

```diff
@@ -81,13 +81,15 @@
     "lint": "npm run lint:js && npm run lint:scss",
     "lint:js": "eslint --ext .js,.jsx .",
     "lint:scss": "stylelint src/**/*.scss",
+    "pack": "webpack -p",
     "precompile": "rm -rf lib",
     "prepare": "npm run compile",
     "prepare-for-release": "terra prepare-for-release",
-    "start": "tt-serve",
-    "start-prod": "tt-serve -p",
-    "start-heroku": "tt-serve-static --port $PORT --site './build' ",
-    "start-static": "npm run compile:prod && tt-serve-static --site ./build",
+    "start": "webpack-dev-server",
+    "start-prod": "webpack-dev-server --env.disableHotReloading -p",
+    "start-heroku": "terra express-server --port '8081' --site ./build",
+    "start-static": "npm run pack && terra express-server --site ./build",
+    "terra": "node node_modules/@cerner/terra-cli/bin/terra",
```

Comment:
What is this line doing? I don't think you should need this as long as terra-cli is installed the terra command should be available to inside the package.json and any npx command.